cardform/src/test/java/com/mercadolibre/android/cardform/tracks/CardFormTrackerTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.mercadolibre.android.cardform.tracks
+
+import com.mercadolibre.android.cardform.di.module.TrackerModule
+import com.mercadolibre.android.cardform.tracks.model.bin.BinClearTrack
+import com.mercadolibre.android.cardform.tracks.model.bin.BinInvalidTrack
+import com.mercadopago.android.px.addons.TrackingBehaviour
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class CardFormTrackerTest {
+
+    private lateinit var cardFormTracker: CardFormTracker
+    private lateinit var trackingBehaviour: TrackingBehaviour
+    private lateinit var trackerModule: TrackerModule
+    private lateinit var track: BinInvalidTrack
+
+    @BeforeEach
+    fun doBefore() {
+        trackingBehaviour = mockk<TrackingBehaviour>(relaxed = true)
+        track = BinInvalidTrack("bintest")
+
+        val trackerData = TrackerData(
+            "siteIdMock",
+            "flowIdMock",
+            "sessionIdMock"
+        )
+
+        cardFormTracker = CardFormTracker(trackerData, trackingBehaviour)
+        trackerModule = TrackerModule("siteIdMock",
+            "flowIdMock",
+            "sessionIdMock",
+            trackingBehaviour
+        )
+
+    }
+
+    @Test
+    fun trackEvent() {
```

Comment:
What do you think about using the GWT pattern that we are adopting in most of the project ?

Martin Fowler's article talking about the topic: https://martinfowler.com/bliki/GivenWhenThen.html

We are also using some features of Junit 5 to better contextualize our tests like the following example:
https://github.com/mercadolibre/card-form-android/blob/master/cardform/src/test/java/com/mercadolibre/android/cardform/presentation/mapper/CardInfoMapperTest.kt